examples/latent_sde.py

```diff
@@ -71,23 +71,47 @@ def f(self, t, y):  # Approximate posterior drift.
         inp = torch.cat((torch.sin(t), torch.cos(t), y), dim=-1)
         return self.net(inp)
 
+    def f_aug(self, t, y):
+        y = y[:, 0][:, None]
```

Comment:
You should be able to simplify this with `y = y[:, 0:1]`.

Reply:
Done!